gui/default/index.html

```diff
@@ -613,9 +613,14 @@ <h4 class="panel-title">
           </div>
         </div>
         <div class="form-group">
-          <button type="button" class="btn btn-sm btn-default pull-right" ng-click="addDevice()">
-            <span class="fa fa-plus"></span>&nbsp;<span translate>Add Remote Device</span>
-          </button>
+          <span class="pull-right">
+            <button type="button" class="btn btn-sm btn-default" ng-click="globalDiskChanges()">
```

Comment:
Yet changelog might be used for displaying syncthing version changes in the future, so not sure if using it would be clever. Activity Log?